Assets/LeapC/StructMarshal.cs

```diff
@@ -16,8 +16,7 @@ private class StructContainer {
     private static int _sizeofT = Marshal.SizeOf(typeof(T));
 
     /**
-     * Converts an IntPtr to a struct of type T.  Does not allocate any
-     * garbage unlike Marshal.PtrToStruct(IntPtr, Type).
+     * Converts an IntPtr to a struct of type T.
```

Comment:
Any reason this comment was changed?  The lack of garbage is a pretty important aspect of its operation.